### PR TITLE
Feature/avatar button header

### PR DESCRIPTION
**Pull request by bulmaroarellano**
## Checklist ✅

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [x] Tests are included
- [x] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

resolves 4

### CODE REVIEW

File: src/components/organisms/CurrencySelector/CurrencySelector.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function CurrencySelector() {
           <CurrencyItem
             flag={flagMX}
             country="México"
-            currency="MXN peso"
+            currency="MXN pesos"
```
Comment (felipedc09) [inline src/components/organisms/CurrencySelector/CurrencySelector.jsx:17]: I think that could be great have a list of currencies or a JSON with all currencies that we want support

File: src/index-popup.js [hunk 2]
```diff
@@ -1,10 +1,146 @@
...
+        lineHeight: '20px',
+        padding:'3px',
+        size:'large',
+        '&:hover':{
+            backgroundPosition:'right center', /* change the direction of the change here */
+        },
+       },
+      }],
+  }},
+});
```
Comment (felipedc09) [inline src/index-popup.js:138]: Could be great have the theme in other file

File: src/styles/customTheme.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        borderRadius: '6px',
+        color: 'white',
+        height: 52,
+        padding: '3px',
+      },
+    },
+  },
+});
+
+return customTheme;
```
Comment (felipedc09) [inline src/styles/customTheme.jsx:97]: You can use this theme file in the popup.js file

**Review by felipedc09 — CHANGES_REQUESTED** [review]

**Review by bulmaroarellano — COMMENTED** [review]
Pending separating the CustomTheme object from the index-popup.js file, I must compare it with some other CustomTheme object from another template, as the MUI documentation is confusing for me

File: src/index-popup.js (hunk 2 above)
Comment (bulmaroarellano) [reply src/index-popup.js:138]: Pending separating the CustomTheme object from the index-popup.js file, I must compare it with some other CustomTheme object from another template, as the MUI documentation is confusing for me

File: src/components/organisms/CurrencySelector/CurrencySelector.jsx (hunk 1 above)
Comment (bulmaroarellano) [reply src/components/organisms/CurrencySelector/CurrencySelector.jsx:17]: Applied, thanks Felipe

**Pull request merged**